lib/suspenders/app_builder.rb

```diff
@@ -300,9 +300,12 @@ def init_git
     end
 
     def create_heroku_apps(flags)
+      staging_environment = "RACK_ENV=staging RAILS_ENV=staging"
```

Comment:
How about naming the variable after the ENV variable like `serve_static_files`? `rack_env = = "RACK_ENV=staging RAILS_ENV=staging"`